src/index.html

```diff
@@ -22,6 +22,455 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="navigation">
+      <div class="navigation__wrapper">
+        <div class="navigation__logo">
+          <img
+            src="images/logo.png"
+            alt="moyo logo"
+          />
+        </div>
+      </div>
+      <nav class="navigation__nav">
+        <ul class="navigation__list">
+          <li class="navigation__item">
+            <a
+              href="#"
+              class="navigation__link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              href="#"
+              class="navigation__link"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              href="#"
+              class="navigation__link"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              href="#"
+              class="navigation__link"
+              data-qa="hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              href="#"
+              class="navigation__link"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              href="#"
+              class="navigation__link"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              href="#"
+              class="navigation__link"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              href="#"
+              class="navigation__link"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <article
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__top">
+          <img
+            src="images/imac.jpeg"
+            class="card__top-img"
+            alt="imac"
+          />
+        </div>
+
+        <div class="card__bottom">
+          <h2 class="card__bottom-tittle">
+            APPLE A1419 iMac 27" Retina
+            <br />
+            5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__bottom-code">Product code: 195434</p>
+
+          <div class="card__botoom-stars">
```

Comment:
There's a typo in the class name 'card__botoom-stars'. It should be 'card__bottom-stars'.